.github/workflows/test.yml

```diff
@@ -77,8 +77,6 @@ jobs:
     - name: Build git-perf
       run: ${{ env.CARGO }} build --verbose ${{ env.TARGET_FLAGS }}
 
```

Comment:
already present on https://github.com/kaihowl/git-perf/blame/7338cb8570d3bacf24817f343518bdde469d2ea9/.github/workflows/test.yml#L56-L57. Keeping the last added one.